crates/bevy_render/src/render_resource/bind_group.rs

```diff
@@ -109,7 +109,7 @@ impl Deref for BindGroup {
 /// * `uniform(BINDING_INDEX)`
 ///     * The field will be converted to a shader-compatible type using the [`ShaderType`] trait, written to a [`Buffer`], and bound as a uniform.
 ///     [`ShaderType`] is implemented for most math types already, such as [`f32`], [`Vec4`](bevy_math::Vec4), and
-///   [`Color`](crate::color::Color). It can also be derived for custom structs.
+///   [`Color`](crate::color::LegacyColor). It can also be derived for custom structs.
```

Comment:
shouldn't we call that `LegacyColor` instead of `Color`?

Reply:
Fixed.